File: root/programs/C#/Samples/WebApp_sample/SPA_Sample/SPA_Sample/Startup.cs
using Microsoft.Owin;
using Owin;

using System.Web.Http;
using System.Web.Mvc;
using System.Web.Optimization;
using System.Web.Routing;


[assembly: OwinStartup(typeof(SPA_Sample.Startup))]

namespace SPA_Sample
{
    public class Startup
    {
        public void Configuration(IAppBuilder app)
        {
            // アプリケーションの設定方法の詳細については、http://go.microsoft.com/fwlink/?LinkID=316888 を参照してください

            AreaRegistration.RegisterAllAreas();

            WebApiConfig.Register(GlobalConfiguration.Configuration);
            FilterConfig.RegisterGlobalFilters(GlobalFilters.Filters);
            RouteConfig.RegisterRoutes(RouteTable.Routes);
            BundleConfig.RegisterBundles(BundleTable.Bundles);
            AuthConfig.RegisterAuth();
        }
    }
}

File: root/programs/VB/Samples/WebApp_sample/WebForms_Sample/WebForms_Sample/Startup.vb
'**********************************************************************************
'* テンプレート
'**********************************************************************************

' サンプル中のテンプレートなので、必要に応じて使用して下さい。

'**********************************************************************************
'* クラス名        ：OwinStartup
'* クラス日本語名  ：OwinStartup
'*
'* 作成日時        ：－
'* 作成者          ：－
'* 更新履歴        ：－
'*
'*  日時        更新者            内容
'*  ----------  ----------------  -------------------------------------------------
'*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
'**********************************************************************************

Imports System.Net.Http
Imports System.Web.Optimization
Imports System.Web.Routing

Imports Owin
Imports Microsoft.Owin

Imports Touryo.Infrastructure.Framework.Authentication

<Assembly: OwinStartup(GetType(WebForms_Sample.Startup))>

Namespace WebForms_Sample
    Public Class Startup
        Public Sub Configuration(app As IAppBuilder)
            ' アプリケーションの設定方法の詳細については、http://go.microsoft.com/fwlink/?LinkID=316888 を参照してください

            ' アプリケーションのスタートアップで実行するコードです

            ' URLルーティングの登録
            RouteConfig.RegisterRoutes(RouteTable.Routes)
            ' バンドル＆ミニフィケーションの登録
            BundleConfig.RegisterBundles(BundleTable.Bundles)

            ' JwkSet取得用
            OAuth2AndOIDCClient.HttpClient = New HttpClient()
        End Sub
    End Class
End Namespace
